cmd/pulumi-language-yaml/language_test.go

```diff
@@ -172,30 +172,15 @@ func runTestingHost(t *testing.T) (string, testingrpc.LanguageTestClient) {
 
 // Add test names here that are expected to fail and the reason why they are failing
 var expectedFailures = map[string]string{
-	"l1-stack-reference":                    "TODO",
-	"l2-destroy":                            "TODO",
-	"l2-invoke-options":                     "TODO",
-	"l2-map-keys":                           "TODO",
-	"l2-provider-grpc-config-schema-secret": "TODO",
-	"l2-resource-config":                    "TODO",
-	"l2-engine-update-options":              "TODO",
-	"l2-invoke-secrets":                     "TODO",
-	"l2-parameterized-resource":             "TODO",
-	"l2-provider-grpc-config-secret":        "TODO",
-	"l2-resource-primitives":                "TODO",
-	"l2-explicit-provider":                  "TODO",
-	"l2-invoke-simple":                      "TODO",
-	"l2-plain":                              "TODO",
-	"l2-ref-ref":                            "TODO",
-	"l2-failed-create-continue-on-error":    "TODO",
-	"l2-invoke-variants":                    "TODO",
-	"l2-primitive-ref":                      "TODO",
-	"l2-resource-alpha":                     "TODO",
-	"l2-target-up-with-new-dependency":      "TODO",
-	"l2-invoke-dependencies":                "TODO",
-	"l2-large-string":                       "TODO",
-	"l2-provider-grpc-config":               "TODO",
-	"l2-resource-asset-archive":             "TODO",
+	"l1-stack-reference":                    "YAML does not support fn::getOutput",
+	"l2-invoke-options":                     "cannot assign expression",
+	"l2-map-keys":                           "missing expected dependency primitive",
+	"l2-provider-grpc-config-schema-secret": "Detected a secret leak in state",
+	"l2-invoke-secrets":                     "YAML does not support fn::secret",
```

Comment:
What does this mean?  YAML has a `fn::secret` here: https://github.com/pulumi/pulumi-yaml/blob/430fa6008462d5bb6f39094a7059c9721cc149e1/pkg/pulumiyaml/ast/expr.go#L782.  Is this different from what we need?  Can we describe that a bit more to avoid confusing others as well?

Reply:
That's just the error message I got when running this test, I didn't dig into it deeper.